spec/requests/ticity_searches_request_spec.rb

```diff
@@ -1,5 +1,136 @@
 require 'rails_helper'
 
 RSpec.describe "TicitySearches", type: :request do
+    let(:user)  {User.create(
+        email: "testover9000@test.com", 
+        password: "123123", 
+        password_confirmation: "123123"
+        )}
+    let(:testSearch) {TicitySearch.create(
+        city1: "Seattle", 
+        city2: "San Diego", 
+        city3: "Austin", 
+        housing: 4, 
+        costofliving: 4, 
+        startups: 4, 
+        capital: 4, 
+        travel: 4, 
+        commute: 4, 
+        business: 4, 
+        safety: 4, 
+        healthcare: 4, 
+        education: 4, 
+        environment: 4, 
+        economy: 4, 
+        taxation: 4, 
+        internetaccess: 4, 
+        leisure: 4, 
+        outdoors: 4, 
+        tolerance: 4, 
+        user_id: user.id
+    )}
 
+    describe "index" do
+        it "gets a list of searches" do
+            testSearch
```

Comment:
Optional: If the purpose of this line is to make sure that testSearch exists, you could change line 9 to use the `let!()` method instead of `let()` to avoid the need for this.